db/python/tables/bq/generic_bq_filter_model.py

```diff
@@ -0,0 +1,109 @@
+import dataclasses
+from typing import Any
+
+from db.python.tables.bq.generic_bq_filter import GenericBQFilter
+from db.python.utils import GenericFilterModel
+
+
+def prepare_bq_query_from_dict_field(
+    filter_, field_name, column_name
+) -> tuple[list[str], dict[str, Any]]:
+    """
+    Prepare a SQL query from a dict field, which is a dict of GenericFilters.
+    Usually this is a JSON field in the database that we want to query on.
+    """
+    conditionals: list[str] = []
+    values: dict[str, Any] = {}
+    for key, value in filter_.items():
+        if not isinstance(value, GenericBQFilter):
+            raise ValueError(f'Filter {field_name} must be a GenericFilter')
+        if '"' in key:
```

Comment:
Might also want to check for a single quote `'`

Reply:
Ok, added check for '